.coveragerc

```diff
@@ -0,0 +1,32 @@
+# .coveragerc to control coverage.py
+[run]
+branch = True
+
+[report]
+# Regexes for lines to exclude from consideration
+exclude_lines =
+	# Have to re-enable the standard pragma
+	pragma: no cover
+	
+	# Don't complain about missing debug-only code:
+	#def __repr__
+	#if self\.debug
+	
+	# Don't complain if tests don't hit defensive assertion code:
+	#raise AssertionError
+	raise NotImplementedError
+	except ImportError
+	except TypeError
+	except ArchiveConnectionError
+	raise httpexceptions
+	except KeyError
+	except Exception as e:
```

Comment:
No problem.  I'll make sure that the other error handling lines not directly relevant to this pull request are included in the coveralls test.  But they're still not currently being tested at them moment.  We should probably add an issue to make sure there are tests to cover these error handling cases in the future.